tensorflow/compiler/tf2xla/cpu_function_runtime.h

```diff
@@ -104,7 +104,7 @@ class BufferInfo {
  private:
   BufferInfo() = default;
 
-  enum class Kind : unsigned {
```

Comment:
Please add a comment here stating why this needs to be `uint64`.

Reply:
See #26958. The `assert` in the function `std::pair<uint64, uint64> Encode() const` throws an error on visual studio compiler because of different sizes of `Kind`. This commit fixes the error. And I saw on `master` branch it's done the same way.

Reply:
I meant on the code itself. :)  The change itself looks fine.